src/ig/IGExporter.ts

```diff
@@ -426,95 +427,78 @@ export class IGExporter {
       'vocabulary',
       'examples' // Must come last in case examples are of other resources
```

Comment:
Technically this comment isn't true anymore, right?  Since we preload into FHIRDefs, `examples` can be processed _whenever_ here...

Reply:
Yeah those comments are no longer relevant. I will remove them.